src/main/java/org/terasology/fluid/component/FluidContainerItemComponent.java

```diff
@@ -22,18 +22,31 @@
 import org.terasology.rendering.assets.texture.TextureRegionAsset;
 
 /**
- * @author Marcin Sciesinski <marcins78@gmail.com>
+ * This component indicates that an entity is a container capable of holding fluids.
  */
 public class FluidContainerItemComponent implements Component, ItemDifferentiating {
+    /** The type of the fluid it can contain */
     @Replicate
     public String fluidType;
```

Comment:
Need a newline after every field here, i.e.

```java
/** The type of the fluid it can contain */
@Replicate
public String fluidType;

/** The current volume of fluid in the container */
public float volume;

/** The maximum volume ... */
```